### DISCUSSION TITLE
generel config, an "include file option" would be helpful

### DISCUSSION THREAD

**Discussion by StefanSa** [vectordotdev/vector#10873] [General]
hi there,
have this structure under `/etc/vector` 
```
/etc/vector/modules
               └───sophos-xg/*.toml
               └───sophos-utm/*.toml
               └───cisco-asa/*.toml
               └───cisco-ios/*.toml
               └───juniper/*.toml
```
and start vector as follows:
`vector -c /etc/vector/vector.toml -c /etc/vector/modules/*/*.toml`

it would be helpful if this could be simplified, with this option `Include` e.g. in the `vector.toml`.
```
# Change this to use a non-default directory for Vector data storage:
# data_dir = "/var/lib/vector"

[api]
enabled = true

Include /etc/vector/modules/*/*.toml
```
With this option you don't need to customize the services of Linux / Windows after the installation,
because you can pass the config paths via `Include` in the *.toml file.
Would something like this be possible ?

**Comment by jszwedko** [comment]
Hi @StefanSa ! I think this is covered by https://github.com/vectordotdev/vector/issues/6789 . What do you think?

**Reply by StefanSa** [reply]
Hi @jszwedko 
Yes i also think that this is covered Jesse.
When can we expect the realization here ?

**Reply by jszwedko** [reply]
Hi @StefanSa ! It is not currently scheduled so unfortunately I can't provide a timeline.
